extensions/amp-ad-exit/0.1/config.js

```diff
@@ -138,3 +148,16 @@ function assertTarget(name, target, config) {
     }
   }
 }
+
+/**
+ * Checks whether a vendor is valid (i.e. listed in vendors.js and has
+ * transport/iframe defined.
+ * @param {string} vendor The vendor name that should be listed in vendors.js
+ */
+export function assertVendor(vendor) {
```

Comment:
Note that this function is used in both this PR, and the other one (#11462). Even though in this PR it's only used in amp-ad-exit.js, suggesting to move it there and make it private...in the other PR it is also used in elsewhere in this file.